images/build.sh

```diff
@@ -30,7 +30,18 @@ if [[ -z "${DOCKER_TAG:-}" ]]; then
     exit 1
 fi
 
-docker_build() {
+if [[ -z "${CONTAINER_ENGINE:-}" ]]; then
+    if  [[ -x `which docker` ]]; then
+        CONTAINER_ENGINE=docker
+    elif [[ -x `which podman` ]]; then
+        CONTAINER_ENGINE=podman
+    else
```

Comment:
fix indent here

also let's leave the default unchanged

Reply:
I fixed the indent.
But what do you mean about "leave the default unchanged"?
Since docker is the first verification, it will become the default if the variable is unset.